src/components/PanelV2/__tests__/PanelV2.spec.js

```diff
@@ -3,38 +3,121 @@
  * @copyright IBM Security 2019
  */
 
-import { mount } from 'enzyme';
-import React, { Fragment } from 'react';
-
-import { label, onClick } from '../_mocks_';
+import { cleanup, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
 
 import { Button, PanelV2, PanelContent } from '../../..';
 
 describe('PanelV2', () => {
-  it('renders', () => {
-    const wrapper = mount(
+  afterEach(cleanup);
+
+  test('should render as expected', async () => {
+    render(
       <PanelV2
-        title={label}
-        subtitle={label}
-        closeButton={{ onClick }}
-        renderFooter={() => (
-          <Fragment>
-            <Button
-              id="example-secondary-button"
-              kind="secondary"
-              onClick={onClick}
-            >
-              Close
-            </Button>
-            <Button id="example-primary-button" onClick={onClick}>
-              Add
-            </Button>
-          </Fragment>
-        )}
+        title="test title"
+        subtitle="test subtitle"
+        closeButton={{
+          label: 'test close',
+        }}
+        renderFooter={() => <Button>test footer button</Button>}
       >
-        <PanelContent>{label}</PanelContent>
+        <PanelContent>test content</PanelContent>
       </PanelV2>
     );
-    expect(wrapper).toMatchSnapshot();
+    await expect(document.body).toMatchSnapshot();
+  });
+
+  test('should have no Axe or DAP violations', async () => {
+    const main = document.createElement('main');
+    render(
+      <PanelV2
+        title="test title"
+        subtitle="test subtitle"
+        closeButton={{
+          label: 'test close',
+        }}
+        renderFooter={() => <Button>test footer button</Button>}
+      >
+        <PanelContent>test content</PanelContent>
+      </PanelV2>,
+      {
+        // DAP requires a landmark '<main>' in the DOM:
+        container: document.body.appendChild(main),
+      }
+    );
+    await expect(document.body).toHaveNoAxeViolations();
+    await expect(document.body).toHaveNoDAPViolations('PanelV2');
```

Comment:
I'm not familiar with this assertion what does this line mean?

Reply:
I added Axe and DAP matchers in this PR, based on similar matchers Carbon added a month or so ago: https://github.com/carbon-design-system/ibm-security/pull/312
Carbon's matchers: https://github.com/carbon-design-system/carbon/tree/ac14f81df750dde81d125e6a2a036e476723b5b9/tasks/jest/matchers

Reply:
Oh, I didn't catch that they were two different assertions. Nice! Looks useful.